source/voice_control/about_wake_word.markdown

```diff
@@ -1,28 +1,25 @@
 ---
-title: "About wake words"
+title: "The Home Assistant approach to wake words "
```

Comment:
This is our old approach to wake words. Should it be named this ?

Reply:
old: openWakeWord, processing done inside HA
new: microWakeWord, processing done on satellite

Reply:
it was formerly named like that when linked, but if its the old approach I'll check with JLo because maybe we don't even want this doc linked this high and be renamed to be sure its known its old